### ISSUE TITLE
Input Types cannot define resolvers

### ISSUE THREAD

**Issue by michaellee8** [Glavin001/graphql-sequelize-crud#28]
When I am using this library in latest `graphql`, I can see the below error
```
CreateusersValuesInput.userId field type has a resolve property, but Input Types cannot define resolvers.
at invariant (/home/michael/revisioner-graphql/node_modules/graphql/jsutils/invariant.js:19:11)
```
but this problem can be eliminated (with possible problems) by falling back to `graphql@^0.8.0`
I believe this is due to a breaking API change, how do you guys think of it

**Comment by Glavin001** [comment]
Please provide some code to reproduce this error.

**Comment by Glavin001** [comment]
I am also experiencing this now with:

```
    "graphql": "^0.11.7",
```

Here are my logs:

```
Unhandled rejection Error: createUserInput.id field type has a resolve property, but Input Types cannot define resolvers.
    at invariant (/Users/glavin/Development/HealthForms/packages/api/node_modules/graphql/jsutils/invariant.js:18:11)
    at /Users/glavin/Development/HealthForms/packages/api/node_modules/graphql/type/definition.js:698:58
    at Array.forEach (native)
    at GraphQLInputObjectType._defineFieldMap (/Users/glavin/Development/HealthForms/packages/api/node_modules/graphql/type/definition.js:692:16)
    at GraphQLInputObjectType.getFields (/Users/glavin/Development/HealthForms/packages/api/node_modules/graphql/type/definition.js:681:49)
    at typeMapReducer (/Users/glavin/Development/HealthForms/packages/api/node_modules/graphql/type/schema.js:223:26)
    at typeMapReducer (/Users/glavin/Development/HealthForms/packages/api/node_modules/graphql/type/schema.js:189:12)
    at Array.reduce (native)
    at /Users/glavin/Development/HealthForms/packages/api/node_modules/graphql/type/schema.js:216:36
    at Array.forEach (native)
    at typeMapReducer (/Users/glavin/Development/HealthForms/packages/api/node_modules/graphql/type/schema.js:209:27)
    at Array.reduce (native)
    at new GraphQLSchema (/Users/glavin/Development/HealthForms/packages/api/node_modules/graphql/type/schema.js:97:34)
    at Object.getSchema (/Users/glavin/Development/HealthForms/packages/api/node_modules/graphql-sequelize-crud/lib/index.js:887:10)
    at /Users/glavin/Development/HealthForms/packages/api/dist/index.js:116:43
    at tryCatcher (/Users/glavin/Development/HealthForms/packages/api/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/glavin/Development/HealthForms/packages/api/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/Users/glavin/Development/HealthForms/packages/api/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/Users/glavin/Development/HealthForms/packages/api/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/Users/glavin/Development/HealthForms/packages/api/node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (/Users/glavin/Development/HealthForms/packages/api/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/Users/glavin/Development/HealthForms/packages/api/node_modules/bluebird/js/release/async.js:143:10)
```

**Comment by Glavin001** [comment]
Duplicate of https://github.com/Glavin001/graphql-sequelize-crud/issues/19

**Comment by Glavin001** [comment]
Published a big new release, v0.4.1! https://github.com/Glavin001/graphql-sequelize-crud/releases/tag/v0.4.1
